test/libs/045-parse_emails.coffee

```diff
@@ -0,0 +1,54 @@
+CONFIG = require 'config'
+__ = CONFIG.universalPath
+_ = __.require 'builders', 'utils'
+
+should = require 'should'
+parseEmails = __.require 'controllers', 'invitations/lib/parse_emails'
+userEmail = 'bla@bla.bla'
+
+describe 'parse emails', ->
```

Comment:
maybe picky, but i dont get the context. When is this `parseEmails` called? for what purpose, is it a validation before the mailer sends it? would be cool to describe a bit more